<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DHSampleDetail</class>
 <widget class="QWidget" name="DHSampleDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1148</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: none;
background: none;

background-color: rgba(0, 0, 0, 120);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>1080</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: none;
border-image: none;
background-image: none;

background: #FFFFFF;
border-radius: 20px;</string>
     </property>
     <widget class="QLabel" name="label_name">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;

width: 46px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #ffffff;
line-height: 34px;</string>
      </property>
      <property name="text">
       <string>详情</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>1021</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>XXX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lab_batchNum">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>XXX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lab_bottleNum">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>XXX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lab_product">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>XXX</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lab_validity">
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(221, 255, 240);</string>
            </property>
            <row>
             <property name="text">
              <string>新建行</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>新建列</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_no">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>500</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(244, 96, 117, 255), stop:1 rgba(201, 67, 86, 255));
border-radius: 10px;
font-weight: bold;
color: white;
font-size: 32px;
font-family: PingFang SC;</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_yes">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>500</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #22A4B3;
border-radius: 10px;

font-family: PingFang SC;
font-weight: bold;
font-size: 32px;
color: #FFFFFF;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
image: none;

background: #22a4b3;

border-top-left-radius: 20px;
border-top-right-radius: 20px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
      </property>
     </widget>
     <zorder>widget_5</zorder>
     <zorder>label_name</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>btn_no</zorder>
     <zorder>btn_yes</zorder>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
